config/checker_guideline_map.json

```diff
@@ -24,10 +21,7 @@
     "alpha.cplusplus.DeleteWithNonVirtualDtor": {
       "sei-cert": ["oop52-cpp"]
     },
-    "alpha.cplusplus.InvalidatedIterator": {
-      "sei-cert": ["ctr51-cpp"]
-    },
-    "alpha.cplusplus.PlacementNew": {
+    "cplusplus.PlacementNew": {
```

Comment:
Please try to keep the key attributes of this config file in alphabetical order. So move this entry under this line:
https://github.com/Ericsson/codechecker/blob/d15d74691a9d30f3825c9e2f20f81051f64db715/config/checker_guideline_map.json#L208